docker-compose.build.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+services:
+  ## This disables wharf-web, which lets you run wharf-web locally
+  ## via `npm start` instead. Comment this out to reenable it
```

Comment:
> Comment this out to reenable it

This confuses me, how would just commenting that section out enable the web? Wouldn't you also need to uncomment this?

![image](https://user-images.githubusercontent.com/22037089/120208337-a73c5d00-c22d-11eb-9b8e-78d79ff853f1.png)

Reply:
Yes, these comments should be revisited. One should be outcommented and one should be commented.

Don't know why there's even 2 `web` sections. Should just be the 1

Will fix

Reply:
Fixed. The comment is IMO much clearer now